UI/src/view/SpeedLimitLayer.js

```diff
@@ -574,9 +700,20 @@ window.SpeedLimitLayer = function(params) {
     });
   };
 
-  var reset = function() {
+  var reset = function(showHistorySpeedLimits) {
+    var vectorLayerToUse;
+    var styleMapToUse;
+
+    if ((typeof showHistorySpeedLimits != 'undefined') && showHistorySpeedLimits === 'true') {
```

Comment:
The only place i found where we are calling the reset function we are not passing any parameter, so i belive it will be always 'undefined' at this condition.